astropy/io/fits/tests/test_connect.py

```diff
@@ -333,6 +333,24 @@ def test_character_as_bytes(self, tmp_path, memmap):
         del t1, t2, t3
         gc.collect()
 
+    @pytest.mark.parametrize("character_as_bytes", (False, True))
+    def test_strip_spaces(self, tmp_path, character_as_bytes):
+        filename = get_pkg_data_filename("data/tb.fits")
+        t = Table.read(
+            filename, character_as_bytes=character_as_bytes, strip_spaces=True
+        )
+        assert t["c2"].tolist() == ["abc", "xy"]
+
+        t = Table.read(
+            filename, character_as_bytes=character_as_bytes, strip_spaces=False
```

Comment:
Unless this is explicitly tested elsewhere, this test should not specify `strip_spaces=False` in order that the current *default* behavior is tested. Then when the default in the function is changed, this test should initially fail.

Reply:
Ok, done.